new_words_spell_checker.txt

```diff
@@ -57,3 +57,4 @@ Soltan
 systemd
 Tosta
 Zeestraten
+Heitor
```

Comment:
This is not needed anymore, Jaime's PR #75 adds it. Could you remove and rebase, please?

```suggestion
```